src/themes/mat/variables.js

```diff
@@ -1,5 +1,5 @@
 export default {
-	mainFont: '"Roboto","Helvetica",Helvetica,Arial,sans-serif',
+	mainFont: '"Cabin", "Roboto","Helvetica",Helvetica,Arial,sans-serif',
```

Comment:
Please confirm that what you want to do is to change the body text away from Roboto

Reply:
Yuuuuup!